raw-data/UW/uw_inventories.tsv

```diff
@@ -130,3 +130,65 @@ berry1999abun.pdf	abun1252	Abun	Ye	p		p
 				ŋ		ŋ		
 				˦		˦		
 				˨		˨		
+
+Heath Donno So grammar 2016.pdf	donn1239	Donno So Dogon	Wendegele	p	"Voiceless stops occur stem-initially... They do not occur medially or finally in native Dogon vocabulary."	p		
```

Comment:
@bnk7 -- looks great in general, but please match the filename to the bibtex key for easy look up, e.g. rename this field heath_donno_so_grammar_2016.pdf and also the PDF in the squibs directory "heath_donno_so_grammar_2016.pdf".